site/docs/patterns/content-status.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Content Status
+layout: DetailPattern
+aliases:
+  - /salt/patterns/content-status
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-content-status--info", label: "Examples" },
+      { href: "https://www.figma.com/file/InTRZY7R7OdeY0Ye9eyzma/(WIP)-Content-Status?type=design&node-id=652-3271&mode=design&t=BoYoZt69VHfzizeO-0", label: "Content Status examples", internal: true },
+    ]
+---
+
+Content Status is a pattern that provides information when content isn’t ready to be shown in a container component of an application.
+
+It remains onscreen until a situation is resolved, or an action is taken by your user, making them a great tool for communication and guidance.
+
+<Diagram
+  src="/img/patterns/content-status/content-status.png"
+  alt="Example showing a content status with an info indicator"
+  border
+/>
+
+We provide design guidance for using Salt [Button](/salt/components/button), [Progress](/salt/components/progress), [Spinner](/salt/components/spinner), [Stack Layout](/salt/components/stack-layout), [Status Indicator](/salt/components/status-indicator), and [Text](/salt/components/text) components to achieve the pattern. You can also find implementation examples in the [Storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-content-status--info).
+
+## When to use it
+
+Use cases for this pattern could include:
+- When content is being loaded
+- When you want to share information with your users
+- When you need to let your users know there’s an error
+- When your user has successfully submitted content
+
+## How to build it
+
+Content Status is comprised of visual status indicators (Progress, Spinner, Status Indicator) and text elements as supporting messages, a Button can be added to reload content or perform a different action if it will help your users. Use Stack Layout to vertically arrange the structure of the components.
+
+<Diagram
+  src="/img/patterns/content-status/build.png"
+  alt="Example showing a content status with components and spacing labeled"
+  border
+/>
+
+<ExampleContainer type="positive" customPillText="Best practices">
+- Make sure it fills the height and width of the container component.
+- Titles and supporting messages should be horizontally and vertically centered in the container.
+</ExampleContainer>
+
+### Write supporting messages
+
+**Have a clear hierarchy:**
+
+Your titles should be stand-alone, revealing key information your users need to know, without relying on the supporting message–which users may skim over
+
+**Be helpful:**
+
+If it’s possible for your users to resolve or rectify an issue, make sure you state this clearly.
+
+**Be simple:** 
+
+Try to limit the length of the title to less than two lines, and supporting messages to no more than three lines.
```

Comment:
Less than two lines? I'm assuming this means 1?

Reply:
Maybe another random one. Does it actually make sense to refer to the number of lines a title spans as length?

Reply:
cc @mikearildbrown

Reply:
```suggestion
Try to limit titles to a single line and supporting messages to no more than three lines.
```
?